common/changes/sayalis-contextMenuWidth_2017-05-23-22-52.json

```diff
@@ -0,0 +1,30 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "added useTargetWidth prop to the context menu, when true it will use the target elemnt's width to render itself",
```

Comment:
My fault for not be clearer about guidelines for this earlier; however, the general standard for these notes is `<affected component>: <short description from the point of view of the consumer of the package`. So, for example, 
```
ContextMenu: added `useTargetWidth` property to match the width of the anchor element when true
```